packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2613,6 +2613,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
         physics: widget.scrollPhysics,
         dragStartBehavior: widget.dragStartBehavior,
         restorationId: widget.restorationId,
+        autoScrollbar: false,
```

Comment:
If this is a multiline textfield, doesn't it need scrollbars to scroll up and down?